### PR TITLE
GPE Product page

### PR DESCRIPTION
**Pull request by sopoour**
## Changes summary
* new custom type `GreenPolicyEvaluatorPage` with respective fields. Each section has its own tab in Prismic.
* created GPE page with all the content
* for the cal.com embed, I've copied the `ModalWithBackdrop.vue` from Bruno's embrace page but with a few modifications to fix some things and also the needed hook `useKeyPress` 
* the cal.com embed is implemented via an embed code using `useHead` on the page

## Testing
* check if all prismic types have fallbacks
* the page itself should be tested to load all prismic content (everything is already published so there is no need to preview but you should see the content instantly on localhost)
* the cal.com embed should be shown when clicking on "Book a demo" and modal should close on backdrop as well as closing button
* overall sanity check of the code and the way how it's written

**PS: it's been already tested by Zak a lot in regards to visuals**

### CODE REVIEW

File: components/Donation.client.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
     >
       <PrismicText
         wrapper="span"
-        :field="donationData.data['donation-button']"
+        :field="donationData.data['donation_button']"
```
Comment (sopoour) [inline components/Donation.client.vue:22]: I've noticed on our website that the donation button has no label as it was wrongly called here

File: slices/TeamMemberSlice/model.json [hunk 2]
```diff
@@ -21,6 +21,24 @@
             "single": "paragraph,preformatted,heading1,heading2,heading3,heading4,heading5,heading6,strong,em,hyperlink,image,embed,list-item,o-list-item,rtl"
           }
         },
+        "department": {
```
Comment (sopoour) [inline slices/TeamMemberSlice/model.json:24]: had to make those additions as I've had to change some types after the teamMemberSlice has been added. But just copied what Pooria had and then there were no conflicts

**Comment by PooriaT** [comment]
Hey @ahmedlamidi @sopoour 

Thanks for your PR.
I checked the FIgma Design and the draft the GRE page. I think that In the section related to "**_What Sets the Green Policy Evaluator Apart?_**", the right side column may need more margin.

File: components/TextWithLogo.vue
[the remark this answers is not in the document]
Comment (sopoour) [reply components/TextWithLogo.vue:1]: That's a really good idea! Lemme do that 🙏  Thanks for your input :)

**Pull request merged**